CDCgov/prime-reportstream
# 12521: OBX to Observation mapping

This PR completes the OBX->Observation->OBX mappings

Test Steps:
1.  Verify the tests pass
2. Verify the mappings https://docs.google.com/spreadsheets/d/1--KYCqp4IKrDO4NyW6ieA69pJylKfnGYHoOTAU7Rkyk/edit#gid=0

## Changes
- Completes the OBX to Observation mapping
- Provides sender and receiver transforms for simple report

## Checklist

### Testing

### Process

## Linked Issues
- #12521 

## To Be Done


## Specific Security-related subjects a reviewer should pay specific attention to
